src/plugins/kibana_react/public/page_layout/page_template.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiEmptyPrompt, EuiPageTemplate, EuiPageTemplateProps } from '@elastic/eui';
+import React, { FunctionComponent } from 'react';
+
+export type KibanaPageTemplateProps = EuiPageTemplateProps & {
+  isEmptyScreen?: boolean;
+  emptyPrompt?: JSX.Element;
```

Comment:
I'd start adding some props comments now so we keep track of what they do. Like:
```suggestion
  /*
   * Disregards any `children` and renders an EuiEmptyPrompt filled with `pageHeader` items centered as the page body.
   */
  isEmptyScreen?: boolean;
  /*
   * Used in conjunction with `isEmptyScreen` to render a custom React element in place of EuiEmptyPrompt.
   */
  emptyPrompt?: JSX.Element;
```